tests/ttnn/unit_tests/test_ternary.py

```diff
@@ -0,0 +1,127 @@
+# SPDX-FileCopyrightText: © 2023-24 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import ttnn
+import pytest
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+class TestEltwiseTernary:
+    def test_ttnn_where(self, input_shapes):
+        device_id = 0
+        device = ttnn.open(device_id)
+
+        torch.manual_seed(0)
+
+        torch_a = torch.rand(input_shapes, dtype=torch.bfloat16)
+        torch_b = torch.rand(input_shapes, dtype=torch.bfloat16)
+        torch_c = torch.rand(input_shapes, dtype=torch.bfloat16)
+
+        a = ttnn.from_torch(torch_a)
+        b = ttnn.from_torch(torch_b)
+        c = ttnn.from_torch(torch_c)
+
+        a = ttnn.to_device(a, device)
+        b = ttnn.to_device(b, device)
+        c = ttnn.to_device(c, device)
+
+        output = ttnn.where(a, b, c)
+        output = ttnn.to_layout(output, ttnn.ROW_MAJOR_LAYOUT)
+
+        print(f"shape: {output.shape}")
```

Comment:
I'm not seeing asserts anywhere on these tests.   It's like the test_<category of op>.py doesn't allow much real testing on these ops.